pkg/sdk/object_types.go

```diff
@@ -204,6 +206,7 @@ const (
 	PluralObjectTypeShares                 PluralObjectType = "SHARES"
 	PluralObjectTypeTables                 PluralObjectType = "TABLES"
 	PluralObjectTypeDynamicTables          PluralObjectType = "DYNAMIC TABLES"
+	PluralObjectTypeCortexSearchServices   PluralObjectType = "CORTEX SEARCH SERVICES"
```

Comment:
Plural form is needed for `show` still.  Is this file only used for grants?

Reply:
It could be used in other contexts, but it is mostly used for grants. So it can be removed, if bulk operations are not supported for cortex.
cc: @sfc-gh-mloring